### ISSUE TITLE
Telegram doesn't work with Wacom drivers installed

### ISSUE THREAD

**Issue by Laxaor** [telegramdesktop/tdesktop#5805]
### Steps to reproduce
1. Install any Wacom driver from [their page](https://www.wacom.com/en/support/product-support/drivers) (tried 6.3.33-3 and 6.3.15-3)
2. Reboot as requested by driver installer
3. Execute telegram.exe

### Expected behaviour
Telegram should open normally

### Actual behaviour
Telegram won't open, nor show in the tray or tool bar.

It does appear on the Task Manager though, with 0% CPU and Network consumption and 1,6 MB of memory used.

Tried reinstalling telegram both before and after the driver instalation, 
made no difference, once you install the driver and reboot telegram stops working.

Logs show nothing beyond startup, and there don't appear to be any errors logged.

### Configuration
**Operating system:**

Windows 10 Home

**Version of Telegram Desktop:**

1.5.15

**Used theme**:

The default Blue theme

<details><summary><b>Logs</b>:</summary><pre><code>
[2019.03.17 11:58:54] Launched version: 1005018, install beta: [TRUE], alpha: 0, debug mode: [FALSE], test dc: [FALSE]
[2019.03.17 11:58:54] Executable dir: C:/Users/Alejandro/AppData/Roaming/Telegram Desktop/, name: Telegram.exe
[2019.03.17 11:58:54] Initial working dir: C:/Users/Alejandro/AppData/Roaming/Telegram Desktop/
[2019.03.17 11:58:54] Working dir: C:/Users/Alejandro/AppData/Roaming/Telegram Desktop/
[2019.03.17 11:58:54] Command line: C:\Users\Alejandro\AppData\Roaming\Telegram Desktop\Telegram.exe -autostart
[2019.03.17 11:58:54] Executable path before check: C:/Users/Alejandro/AppData/Roaming/Telegram Desktop/Telegram.exe
[2019.03.17 11:58:54] Logs started
</code></pre></details>

**Comment by p0358** [comment]
You may want to enable debug mode to enable more verbose logging. By the way, does this happen while you have Wacom device connected or just the driver installed? Can you for example run `driverquery /V | find "wacom"` in cmd and show what it returns? I personally have wacompen.sys installed (but no device connected), and Telegram Desktop works just fine for me

**Comment by Laxaor** [comment]
After reinstalling everything multiple times in different orders, I discovered I had a second instance of wacom drivers for an older device that didn't get removed properly when I uninstalled it. Weirdly though, windows didn't show the drivers or that wacom executable anywhere, so I didn't notice.

Anyways, once I deleted the old driver, everything went smoothly again, so I'm closing the issue since it's pretty clear this isn't a Telegram problem.

Thanks though!
